norse/torch/functional/receptive_field.py

```diff
@@ -7,20 +7,26 @@
 import torch
 
 
-def gaussian_kernel(size: int, c: torch.Tensor, domain: int = 8) -> torch.Tensor:
+def gaussian_kernel(
+    size: int, c: torch.Tensor, x: torch.Tensor, y: torch.Tensor, domain: int = 8
+) -> torch.Tensor:
     """
     Efficiently creates a differentiable 2d gaussian kernel.
 
     Arguments:
       size (int): The size of the kernel
       c (torch.Tensor): A 2x2 covariance matrix describing the eccentricity of the gaussian
+      x (torch.Tensor): The receptive's field center position in x-axis
+      y (torch.Tensor): The receptive's field center position in y-axis
       domain (int): The domain of the kernel. Defaults to 8 (sampling -8 to 8).
     """
     ci = torch.linalg.inv(c)
     cd = torch.linalg.det(c)
     fraction = 1 / (2 * torch.pi * torch.sqrt(cd))
     a = torch.linspace(-domain, domain, size)
     xs, ys = torch.meshgrid(a, a, indexing="xy")
+    xs = xs - x
```

Comment:
This effectively means that a positive x leads to a left-shift. Is that on purpose?
According to the docs, the I would assume a positive x shifts the RF to the right.

Reply:
A positive x leads to a left-shift of the domain, which results in the kernel shifting to the right in the transformed domain. 
A value x<sub>0</sub> performs the transformation g(x) --> g(x-x<sub>0</sub>)

Reply:
Ah, I see. That actually makes sense, thank you!